cores/esp8266/Schedule.h

```diff
@@ -6,20 +6,23 @@
 #define SCHEDULED_FN_MAX_COUNT 32
 #define SCHEDULED_FN_INITIAL_COUNT 4
 
-// Warning 
-// This API is not considered stable. 
-// Function signatures will change.
+// This API was not considered stable but is now fossilizing.
+// Function signatures may change, queue must stay FIFO.
 // You have been warned.
 
-// Run given function next time `loop` function returns, 
+// Run given function ONCE next time `loop` function returns,
+// or `yield` is called,
 // or `run_scheduled_functions` is called.
 // Use std::bind to pass arguments to a function, or call a class member function.
 // Note: there is no mechanism for cancelling scheduled functions.
 // Keep that in mind when binding functions to objects which may have short lifetime.
 // Returns false if the number of scheduled functions exceeds SCHEDULED_FN_MAX_COUNT.
 bool schedule_function(std::function<void(void)> fn);
 
-// Run all scheduled functions. 
+// Run given function every at least <repeat_us> microseconds until it returns false
```

Comment:
The actual behavior is a little different.  I would suggest something like:

`````
// Run given function periodically about every <repeat_us> microseconds until it returns false
// Note that it may be more than <repeat_us> microseconds between calls if `yield` is not called
// frequently, and therefore should not be used for timing critical operations.
````